templates/js/custom-copilot-rag-microsoft365/.tours/load-data-from-graph-connector.tour

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "https://aka.ms/codetour-schema",
+  "title": "Load data from Graph connector",
+  "steps": [
+    {
+      "file": "aad.manifest.json",
+      "description": "### Update Entra app manifest\n\nChange the permission to be able to read external items. This is necessary to grant your app access to external items imported to Microsoft 365 by the Microsoft Graph connector. Change the permission to:\n\n```text\nExternalItem.Read.All\n```",
+      "line": 24,
+      "selection": {
+        "start": {
+          "line": 24,
+          "character": 28
+        },
+        "end": {
+          "line": 24,
+          "character": 42
+        }
+      },
+      "title": "Update Entra app manifest"
+    },
+    {
+      "file": "src/app/app.js",
+      "description": "### Update default scopes\n\nUpdate the scopes that the Microsoft Graph client should request when connecting to Microsoft Graph. This is necessary for your app to be able to read external items imported by the Graph connector. Change the permission to:\n\n```text\nExternalItem.Read.All\n```",
```

Comment:
Our templates have a ts version also, can we create one for ts file?

Reply:
Absolutely! I wanted to start with one to check if we agree with the approach. I'll go on and create the same for the TS template